app/models/suggested_edit.rb

```diff
@@ -0,0 +1,6 @@
+class SuggestedEdit < ApplicationRecord
+  belongs_to :resource, polymorphic: true
+  belongs_to :user
+
```

Comment:
Add `belongs_to :reviewed_by, class_name: 'User'` here to make that an association rather than just an integer.